### PR TITLE
Begin work on Building Markers in street map.

### PR DESCRIPTION
**Pull request by RegDogg**
Closes https://github.com/toontown-archipelago/toontown-archipelago/issues/91

### CODE REVIEW

**Comment by RegDogg** [comment]
Ready to merge, must be merged with https://github.com/toontown-archipelago/panda3d/pull/4

**Review by DarthMDev — CHANGES_REQUESTED** [review]

**Review by DarthMDev — COMMENTED** [review]
some stuff that needs to be looked at

File: toontown/quest/QuestMap.py [hunk 1]
```diff
@@ -139,11 +178,36 @@ def updateQuestInfo(self):
                     if quest.getHolder() == Quests.AnyFish:
                         self.putBuildingMarker(self.fishingSpotInfo, mapIndex=mapIndex)
                     continue
-                elif quest.getType() != Quests.DeliverGagQuest and quest.getType() != Quests.DeliverItemQuest and quest.getType() != Quests.VisitQuest and quest.getType() != Quests.TrackChoiceQuest:
+                elif quest.getType() not in (
```
[the remark this answers is not in the document]
Comment (RegDogg) [reply toontown/quest/QuestMap.py]: Resolved in https://github.com/toontown-archipelago/toontown-archipelago/pull/103/commits/d84ac62d2a77b2599d428e4a8464a0e3e56c5b95

File: toontown/quest/QuestMap.py [hunk 2]
```diff
@@ -112,7 +115,10 @@ def putBuildingMarker(self, pos, mapIndex = None):
         marker['text'] = '%s' % mapIndex
         marker['text_scale'] = 0.7
         marker['image'] = iconNP
-        marker['image_color'] = (1, 0, 0, 1)
+        if isSuitBlock: # Make the bubble appear gray if a cog took over a task building
```
[the remark this answers is not in the document]
Comment (RegDogg) [reply toontown/quest/QuestMap.py]: Resolved in https://github.com/toontown-archipelago/toontown-archipelago/pull/103/commits/d84ac62d2a77b2599d428e4a8464a0e3e56c5b95

**Review by DarthMDev — APPROVED** [review]

**Comment by DarthMDev** [comment]
LGTM!

**Review by RegDogg — COMMENTED** [review]
Final changes made, good to go for review from @DevvyDont

**Review by DevvyDont — APPROVED** [review]
lgtm! will merge this in when new engine build is ready

**Comment by RegDogg** [comment]
Final changes made, ready for merge into https://github.com/toontown-archipelago/toontown-archipelago/tree/feature/cog-building-street-map when ready

**Pull request merged**